content/docs/reference/overview.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Reference
+id: '/docs/reference/overview'
+last_edited: '2022-06-14'
+prev: null
+next: '/docs/reference/types'
+---
+
+## Overview
+
+Tina is a Git-backed headless content management system that enables developers and content creators to collaborate seamlessly. With Tina, developers can create a custom visual editing experience that is perfectly tailored to their site. 
```

Comment:
```suggestion
```

This is the same blurb from our current [Introduction overview](https://tina.io/docs/) right? I'd say we can probably leave it out from this section.